rurusetto/wiki/templates/wiki/edit_ruleset_wiki.html

```diff
@@ -68,6 +68,17 @@ <h1>Icon & Logo</h1>
                     </div>
                   </div>
             </div>
+            <div class="fieldWrapper">
+              <div class="mb-3 row">
+                 <label class="col-sm-2 col-form-label form-label" for="id_icon">{{ form.dark_icon.label_tag }}</label>
```

Comment:
```suggestion
                 <label class="col-sm-2 col-form-label form-label" for="id_icon">{{ form.light_icon.label_tag }}</label>
```
Because the website's default theme is dark so it need to change.